CHANGELOG.md

```diff
@@ -11,7 +11,7 @@ We use *breaking :warning:* to mark changes that are not backward compatible (re
 ## Unreleased
 
 ### Fixed
-
+- [#5893](https://github.com/thanos-io/thanos/pull/5893) Cache: fixed redis client not respecting `SetMultiBatchSize` config value.
```

Comment:
Please add the last line in this section.

Reply:
so just to confirm, the changelog line should be added last to the list of Fixes, is that correct?

Reply:
Yeah, fixed section is good